internal/extension/smartagentextension/factory.go

```diff
@@ -41,18 +42,24 @@ func NewFactory() component.ExtensionFactory {
 }
 
 var bundleDir = func() string {
-	out := os.Getenv(constants.BundleDirEnvVar)
-	if out == "" {
-		exePath, err := os.Executable()
-		if err != nil {
-			panic("Cannot determine agent executable path, cannot continue")
-		}
-		out, err = filepath.Abs(filepath.Join(filepath.Dir(exePath), ".."))
-		if err != nil {
-			panic("Cannot determine absolute path of executable parent dir " + exePath)
+	dir := os.Getenv(constants.BundleDirEnvVar)
+	if dir == "" {
+		if runtime.GOOS == "windows" {
+			dir = filepath.Join("C:\\Program Files", "Splunk", "OpenTelemetry Collector", "agent-bundle")
```

Comment:
I think you'll want to os.Getenv("programfiles") since the default location can vary.

Reply:
Good idea, updated.